### ISSUE TITLE
OcTextarea has blue focus outline

### ISSUE THREAD

**Issue by JammingBen** [opencloud-eu/web#1992]
### Initial Checklist

- [x] I understand this is a bug report and questions should be posted in the [Community Discussions](https://github.com/orgs/opencloud-eu/discussions/)
- [x] I searched [issues](https://github.com/opencloud-eu/web/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Bug Description

The `OcTextinput` component of the design system has a blue focus outline that doesn't match our color roles. It should have the same color as e.g. the `OcTextInput` component.

### Reproduction Steps

- go to https://docs.opencloud.eu/design-system/components/OcTextarea.html
- click inside the textarea form

### Expected Outcome

The outline should match the one from the `OcTextInput` component.

### Actual Outcome

The outline is blue.
